pkg/cmd/pipelinerun/describe.go

```diff
@@ -30,60 +30,60 @@ import (
 	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
-const templ = `Name:	{{ .PipelineRun.Name }}
-Namespace:	{{ .PipelineRun.Namespace }}
+const templ = `{{decorate "bold" "Name"}}:	{{ .PipelineRun.Name }}
+{{decorate "bold" "Namespace"}}:	{{ .PipelineRun.Namespace }}
 {{- $pRefName := pipelineRefExists .PipelineRun.Spec }}{{- if ne $pRefName "" }}
-Pipeline Ref:	{{ $pRefName }}
+{{decorate "bold" "Pipeline Ref"}}:	{{ $pRefName }}
 {{- end }}
 {{- if ne .PipelineRun.Spec.ServiceAccountName "" }}
-Service Account:	{{ .PipelineRun.Spec.ServiceAccountName }}
+{{decorate "bold" "Service Account"}}:	{{ .PipelineRun.Spec.ServiceAccountName }}
 {{- end }}
 
-Status
+{{decorate "underline bold" "Status\n"}}
 STARTED	DURATION	STATUS
```

Comment:
Is an extra space needed here before `STARTED`?

Reply:
Looking at your screenshots helped clarify why. I see now it looks better this way.